Felienne/hedy
# Fixes main container paddings

**Description**
In this PR we make some minor changes to how we handle the `main` block within our templates. We add a bottom padding to the block itself and remove the manual added bottom margin in several templates. This ensures that there is always a nice space between the block content and the bottom of the page.

**Fixes**
This PR fixes #3212.

**How to test**
Log in, navigate to `/landing-page` and verify that the bottom container has a nice space from the bottom of the page.
